tests/sites/mapping/devices/GAT-123/expected/errors.map

```diff
@@ -1,6 +1,7 @@
 Exceptions for GAT-123
   While loading credentials for local device GAT-123
     Credential cloud.auth_type definition missing
+  Cannot read field "min_loglevel" because "this.metadata.system" is null
```

Comment:
I think there's something here that might need some addressing... shouldn't just be exporting a NPE...